sdk/cmake/gcc.cmake

```diff
@@ -54,6 +54,24 @@ if(CMAKE_C_COMPILER_ID STREQUAL "GNU")
     if (DBG)
         add_compile_options("$<$<COMPILE_LANGUAGE:C>:-Wold-style-declaration>")
     endif()
+
+    if(CMAKE_C_COMPILER_VERSION VERSION_GREATER_EQUAL 11)
+        add_compile_options(-fno-builtin-ceil)
+        add_compile_options(-fno-builtin-ceilf)
+        add_compile_options(-fno-builtin-cos)
+        add_compile_options(-fno-builtin-floor)
+        add_compile_options(-fno-builtin-floorf)
+        add_compile_options(-fno-builtin-pow)
+        add_compile_options(-fno-builtin-sin)
+        add_compile_options(-fno-builtin-sincos)
+        add_compile_options(-fno-builtin-sqrt)
+        add_compile_options(-fno-builtin-sqrtf)
+    endif()
+    if(CMAKE_C_COMPILER_VERSION VERSION_GREATER_EQUAL 13)
```

Comment:
Shouldn't the other changes (winpthread, gcc-compat, ...) too depend on GCC version? As in unneeded for RosBE's GCC 8.4.0.